client/src/components/ButtonsBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FunctionComponent } from "react";
+import { Button, Div, Icon } from "atomize";
+import MicIcon from "@material-ui/icons/Mic";
+import MicOffIcon from "@material-ui/icons/MicOff";
+
+type ButtonsBarProps = {
+  settingsCallback: () => void,
+  statusCallback: () => void,
+  muteCallback: () => void,
+  leaveCallback: () => void,
+  userInfoRef: any
+}
+
+export const ButtonsBar: FunctionComponent<ButtonsBarProps> = (props) => {
+  const defaultButtonStyle = {
+    p: "1rem",
+    m: "0.3rem",
+    bg: "none",
+    h: "2.5rem",
+    w: "2.5rem",
+    hoverBg: "#ffffff29",
+    rounded: "circle",
+  };
+  
+  function SettingsButton() {
+    const { settingsCallback } = props;
+    
+    return (
+      <Button title="Settings (,)" {...defaultButtonStyle} onClick={settingsCallback}>
+        <Icon name="SettingsSolid" color="white" size="24px" />
+      </Button>
+    );
+  };
+    
+  function StatusButton() {
```

Comment:
Same as SettingsButton

```suggestion
  const StatusButton = () => {
```